ratelimit/utils.py

```diff
@@ -153,9 +153,14 @@ def is_ratelimited(request, group=None, fn=None, key=None, rate=None,
         count = initial_value
     else:
         if increment:
-            count = cache.incr(cache_key)
+            try:
+                count = cache.incr(cache_key)
+            except ValueError:
+                return False
```

Comment:
both of these early returns won't set the `limited` attribute if the request doesn't already have it.